### ISSUE TITLE
Problem with CBoW

### ISSUE THREAD

**Issue by pnugues** [nzw0301/keras-examples#9]
Hello,
I tried your CBoW program and it did not work. I have the latest versions of Anaconda, Keras, and Tensorflow. My machine is a mac.
1/ When opening the Alice text, Python complains that it is not UTF-8. I had to convert this text to be able to read it.
2/ The shape of the labels is not correct. You can correct it with the statement
labels.append([word])
instead of
labels.append(word)

Thank you for putting your code online.
Pierre

**Comment by nzw0301** [comment]
@pnugues Thank you for your report.
1. You are right. This text file was not utf-8. I use utf-8 text file instead of it. Thanks!
2. I fixed it by another way.
